src/NzbDrone.Core/Datastore/Migration/113_music_blacklist.cs

```diff
@@ -0,0 +1,24 @@
+﻿using FluentMigrator;
+using NzbDrone.Core.Datastore.Migration.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace NzbDrone.Core.Datastore.Migration
+{
+    [Migration(113)]
+    public class music_blacklist : NzbDroneMigrationBase
+    {
+        protected override void MainDbUpgrade()
+        {
+            Alter.Table("Blacklist")
+                  .AddColumn("ArtistId").AsInt32().WithDefaultValue(0)
```

Comment:
Just making sure that these should have a default value rather than being null. It looks like the SQL below allows at least the Artist ID to be null.